src/js/components/Button/index.d.ts

```diff
@@ -38,6 +38,7 @@ export interface ButtonProps {
   href?: string;
   target?: '_self' | '_blank' | '_parent' | '_top' | string;
   icon?: JSX.Element;
+  kind?: string | ButtonType;
```

Comment:
Is this intended to be `ButtonKindType`?

```suggestion
  kind?: string | ButtonKindType;
```

Reply:
Looking at the way we have done it in Pagination, I think it should actually be ButtonType. We can discuss but I am updating import statements to match.